### PR TITLE
chore(ci): bumped perf unit tests threshold to 5% (relative) slowdown.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

Same threshold used for google benchmarks.
Sometimes it fails spuriously with only 1% threshold: https://github.com/falcosecurity/libs/actions/runs/10879002020/job/30182876688?pr=2069

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by Andreagit97 — APPROVED** [review]
/approve

File: .github/workflows/perf.yml [hunk 1]
```diff
@@ -105,11 +105,11 @@ jobs:
 
       # Check will fail if sum of all differences is >= 1%.
       # But we will always comment with the perf diff from master
-      - name: Check >= 1% threshold - perf unit tests
+      - name: Check >= 5% threshold - perf unit tests
```
Comment (leogr) [inline .github/workflows/perf.yml:108]: The comment still says `differences is >= 1%.`

**Review by leogr — APPROVED** [review]
Found a very minor issue. Just nit.

Also, I'm wondering if `5%` is too high. Is there any compelling reason not to choose `2%` or `4%` instead?

cc @jasondellaluce @LucaGuerra

**Comment by FedeDP** [comment]
> Also, I'm wondering if 5% is too high. Is there any compelling reason not to choose 2% or 4% instead?

We used 5% for the benchmarks CI; i think it will save us lots of false positives; having a sensibility of 1 or 2% (relative) is way too hard to reach consistently.

**Pull request merged**